src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -657,7 +657,7 @@ export default {
 	},
 	mounted() {
 		Promise.all([
-			this.fetchRemainingLoanChannels(),
+			// this.fetchRemainingLoanChannels(),
```

Comment:
oops

Reply:
Yup, will fix this up in my final commit when I get the rest of the tracking fixed up.